flang/docs/GettingStarted.md

```diff
@@ -88,6 +88,10 @@ cmake \
 ninja
 ```
 
+On Darwin, to make flang able to link binaries with the default sysroot without
+having to specify additional flags, use the `DEFAULT_SYSROOT` cmake flag, e.g.
```

Comment:
```suggestion
having to specify additional flags, use the `DEFAULT_SYSROOT` CMake flag, e.g.
```

Reply:
I missed this one. Fixed on a follow up commit.